django_docker_engine/container_managers/docker_engine.py

```diff
@@ -117,7 +122,14 @@ def get_url(self, container_name):
             host = 'localhost'
         else:  # pragma: no cover
             raise RuntimeError('Unexpected base_url: %s', self._base_url)
-        container = self._containers_client.get(container_name)
+
+        try:
+            container = self._containers_client.get(container_name)
+        except ReadTimeout:
```

Comment:
Does the exception have any information about how long we waited? Not sure it would be that useful in any case.